tests/agent_unittests/test_agent_protocol.py

```diff
@@ -80,6 +80,11 @@ def send_request(
         request = Request(method=method, path=path, params=params, headers=headers, payload=payload)
         self.SENT.append(request)
         if self.STATUS_CODE:
+            # Define behavior for a 200 status code for use in test_super_agent_health.py
+            if self.STATUS_CODE == 200:
+                payload = {"return_value": "Hello World!"}
+                response_data = json_encode(payload).encode("utf-8")
+                return self.STATUS_CODE, response_data
```

Comment:
This thing should probably be moved to `testing_support/` if it's getting reused in 2 different test suites now.